### PR TITLE
More oneplus and oplus apps

### PR DESCRIPTION
**Pull request by Application-maker**

### CODE REVIEW

File: oem.json [hunk 1]
```diff
@@ -4042,6 +4042,12 @@
         "description": "Atlas Service\nSeems to be a Sound provider. Performs a lot of actions upon audio related events. Still not clear if it is really necessary. Runs at boot and stay in background. Pithus analysis: https://beta.pithus.org/report/6d0f9433431cd34a8e9aaef99b329b3623118a1699033be36032f64653dab3d0",
         "removal": "replace"
     },
+    {
+        "id": "com.oplus.camera",
+        "label": "Camera",
+        "description": "Stock Oppo/Oneplus camera app",
+        "removal": "replace"
```
Comment (MuntashirAkon) [inline oem.json:4049]: Set `suggestion` ID to `cameras`

File: oem.json [hunk 2]
```diff
@@ -4061,6 +4067,12 @@
         "description": "Sends system failure data to developers. Automatically runs at boot.\n\nPithus analysis: https://beta.pithus.org/report/6031048af7434e9cfe3435244dd105ac70e3bfe1f25ecdcca9b2a40b356590a2",
         "removal": "delete"
     },
+    {
+        "id": "com.oplus.deepthinker",
+        "label": "Intelligent Services",
+        "description": "No clue what is it for, acesses location",
```
Comment (MuntashirAkon) [inline oem.json:4073]: It looks like you haven't done any investigation. A simple search returned the following results:
1. https://www.reddit.com/r/Realme/comments/u7rfgs/unnecessary_optimize_battery_notifications/
2. https://forum.xda-developers.com/t/app-optimisation-notification.4423843/
3. https://github.com/melontini/realme-app-list#intelligent-services

They suggest that it definitely breaks something and what purpose might this app be serving.

File: oem.json [hunk 3]
```diff
@@ -4106,6 +4118,24 @@
...
+    {
+        "id": "com.oplus.qualityprotect",
+        "label": "QualityProtect",
+        "description": "Probably Oppo/OnePlus analytics",
+        "removal": "delete"
+    },
+    {
+        "id": "com.oplus.safecenter",
+        "label": "Security center",
+        "description": "Don't know what is it for. Every security function works without it",
```
Comment (MuntashirAkon) [inline oem.json:4130]: Breaks app lock feature according to this post: https://forum.xda-developers.com/t/the-oneplus-10-pro-debloat-thread.4503969/page-2#post-87920315

**Review by MuntashirAkon — CHANGES_REQUESTED** [review]
Does not appear to be well investigated. Please try to investigate a bit and include interesting web links as references. Just because deleting those packages did not affect you does not mean that they wouldn't affect others. There's always a possibility that you've missed something.

**Comment by shuvashish76** [comment]
1. If it's from a single device, try to add all packages in a single PR/commit. (not important)
2. Basic (whoogle search) investigation & test on your device required.
3. If (2) doesn't match to your personal investigation or creates confusion (e.g - online investigation suggests it causes bootloop or breaks something but no such issues found in your personal test) then better add them to `pending` list. Pending list requires only minimal information.
4. In case you're not comfortable to disable/remove a package for testing because e.g - it might create bootloop or you're simply lazy for basic investigation, then add it to `pending` list.

These are not official repo criteria.***

File: oem.json (hunk 2 above)
Comment (Application-maker) [reply oem.json]: for me it doesn't break anything

**Comment by MuntashirAkon** [comment]
Please fix the syntax issues before requesting for a review. Syntaxes are described in the docs.

File: pending.json [hunk 4]
```diff
@@ -419,6 +419,24 @@
         "description": "Probably related to SMS based on the name?\nContains no services and I've never seen it run.",
         "removal": "caution"
     },
+    {
+        "id": "com.oplus.aod",
+        "label": "Aod",
+        "description": "Multiple sources say that AOD doesnt work without tons of extra services, but on some devices it works without extra services",
```
Comment (MuntashirAkon) [inline pending.json:425]: I didn't understand the description. What does _extra services_ mean?

Comment (Application-maker) [reply pending.json:425]: By *extra services* I mean other system apps

**Review by MuntashirAkon — APPROVED** [review]

**Pull request merged**